src/avram/slugify.cr

```diff
@@ -49,6 +64,22 @@ module Avram::Slugify
     end
   end
 
+  def generate(using slug_candidate : Avram::Attribute(String) | String) : String
+    generate([slug_candidate])
+  end
+
+  def generate(using slug_candidates : Array(String | Avram::Attribute(String) | Array(Avram::Attribute(String)))) : String
+    slug_candidates = format_candidates(slug_candidates)
+
+    slug_candidates.first? || UUID.random.to_s
```

Comment:
A uuid fallback would not be my expectation here, but I'm not sure what would have been my expectation. A slug is usually designed to be more "human friendly" than a UUID... so I think I'd just expect it to raise if it couldn't build a slug.